composer.json

```diff
@@ -34,6 +35,9 @@
 		"vendor-dir": "lib/vendor",
 		"platform": {
 			"php": "7.3.0"
+		},
```

Comment:
This addition seems spurious to this feature.  I think it was probably backported from `main` inadvertently.

Reply:
The allow-plugins is because when running `composer install`, it complains that `cweagans/composer-patches contains a Composer plugin which is currently not in your allow-plugins config. ` and I must have hit y.

Reply:
Actually, if it isn't allowed then it won't build properly and will throw an error: 
`PHP Fatal error:  Cannot redeclare __() (previously
 declared in /var/www/ojs-stable-3_3_0/lib/pkp/lib/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php:893) in /var/www/ojs-
stable-3_3_0/lib/pkp/classes/i18n/PKPLocale.inc.php on line 921`

Reply:
This was added in `main` here: https://github.com/pkp/pkp-lib/commit/8f9d32a8ed5157ebc8c467f03c9f6ad8940f14f2

And was discussed and affirmed in Slack, though not with respect to 3.3.

I'll allow it, though pending the broader conversation on lockfile dependency management.